indolaw-nextjs/components/PrimitiveStructure.tsx

```diff
@@ -59,7 +63,9 @@ function maybeLinkToOtherLaws(text: string): string | Array<JSX.Element> {
             key={i}
             target="_blank"
             rel="noopener noreferrer"
-            href={'https://search.hukumonline.com/search/all/?q=undang+undang+13+2003&language=%5B%22id%22%5D'}
+            href={
+              "https://search.hukumonline.com/search/all/?q=undang+undang+13+2003&language=%5B%22id%22%5D"
+            }
```

Comment:
Wait why is this liking only to UU 13 2003 hahaha

And we're not using this function anymore right?